app/models/service_object.rb

```diff
@@ -63,4 +63,14 @@ def validate_key(*keys)
   def default(key, value)
     context.send("#{key}=", value) unless context.send(key).present?
   end
+
+  def something_else
+    if true
+      return "foo"
+    elsif false
+      return "bar"
+    elsif "testing code style"
```

Comment:
Lint/LiteralInCondition: Literal `"testing code style"` appeared in a condition.